docs/DeveloperGuide.md

```diff
@@ -155,6 +155,56 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### View Person Feature
+
+#### Implementation
+
+To make the GUI cleaner and simpler, we decided not to display the schedule of each person in the person list.
```

Comment:
Would this part be better placed under design considerations rather than implementation?

Reply:
I agree with Yan Han on this point.